lib/services/ngsi/entities-NGSI-v1.js

```diff
@@ -243,7 +243,7 @@ function sendUpdateValueNgsi1(entityName, attributes, typeInformation, token, ca
     if (
         'autoprovision' in typeInformation &&
         /* jshint -W101 */
-        typeInformation.autoprovision === undefined
+        typeInformation.autoprovision !== undefined
```

Comment:
The same typo is not happening in lib/services/ngsi/entities-NGSI-v2.js file?

Just to be sure... :)

Reply:
It seems that it's ok:
https://github.com/kandarp-n/iotagent-node-lib/blob/8cc5aa4f5fcc8501d8e24e564ff425de91d814a6/lib/services/ngsi/entities-NGSI-v2.js#L403-L406

Reply:
Great. NTC